src/mui/button/SaveButton.js

```diff
@@ -1,26 +1,30 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
+import { submit } from 'redux-form';
 import RaisedButton from 'material-ui/RaisedButton';
 import FlatButton from 'material-ui/FlatButton';
 import ContentSave from 'material-ui/svg-icons/content/save';
 import CircularProgress from 'material-ui/CircularProgress';
 import translate from '../../i18n/translate';
 
-class SaveButton extends Component {
+export class SaveButton extends Component {
 
     handleClick = (e) => {
         if (this.props.saving) {
             // prevent double submission
             e.preventDefault();
+        } else if (!this.props.submitOnEnter) {
```

Comment:
if `submitOnEnter === true` the button type is `submit`, see line 24. So it falls back to how it used to work. We don't need to explicitly dispatch a `submit` action.

Should I add a comment there? This might not be obvious at first sight. I had to think about it too.